### PR TITLE
chore(typos): fix typos

### PR DESCRIPTION
**Pull request by stavares843**
Fix typos across codebase.

### CODE REVIEW

File: bench/snippets/_fixture.txt [hunk 1]
```diff
@@ -93,7 +93,7 @@
...
+  <meta name="octolytics-dimension-user_id" content="2625545" /><meta name="octolytics-dimension-user_login" content="botika" /><meta name="octolytics-dimension-repository_id" content="160065385" /><meta name="octolytics-dimension-repository_now" content="botika/v_escape" /><meta name="octolytics-dimension-repository_public" content="true" /><meta name="octolytics-dimension-repository_is_fork" content="false" /><meta name="octolytics-dimension-repository_network_root_id" content="160065385" /><meta name="octolytics-dimension-repository_network_root_now" content="botika/v_escape" /><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false" />
```
Comment (stavares843) [inline bench/snippets/_fixture.txt]:
```suggestion
  <meta name="octolytics-dimension-user_id" content="2625545" /><meta name="octolytics-dimension-user_login" content="botika" /><meta name="octolytics-dimension-repository_id" content="160065385" /><meta name="octolytics-dimension-repository_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_public" content="true" /><meta name="octolytics-dimension-repository_is_fork" content="false" /><meta name="octolytics-dimension-repository_network_root_id" content="160065385" /><meta name="octolytics-dimension-repository_network_root_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false" />```

File: bench/snippets/_fixture.txt [hunk 2]
```diff
@@ -1200,7 +1200,7 @@
       </tr>
       <tr>
         <td id="L101" class="blob-num js-line-number" data-line-number="101"></td>
-        <td id="LC101" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    venenatis. Vivamus non dolor hendrerit, vulputate velit sed, varius nunc. Quisque</span></td>
+        <td id="LC101" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    venenatis. Vivamus non dolor hendrerit, vulputate velit sed, various nunc. Quisque</span></td>
```
Comment (stavares843) [inline bench/snippets/_fixture.txt]:
```suggestion
        <td id="LC101" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    venenatis. Vivamus non dolor hendrerit, vulputate velit sed, varius nunc. Quisque</span></td>

File: bench/snippets/_fixture.txt [hunk 3]
```diff
@@ -1284,7 +1284,7 @@
       </tr>
       <tr>
         <td id="L122" class="blob-num js-line-number" data-line-number="122"></td>
-        <td id="LC122" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    non egestas vitae, hendrerit sed quam. Phasellus a nunc eu erat varius imperdiet. Etiam id</span></td>
+        <td id="LC122" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    non egestas vitae, hendrerit sed quam. Phasellus a nunc eu erat various imperdiet. Etiam id</span></td>
```
Comment (stavares843) [inline bench/snippets/_fixture.txt]:
```suggestion
        <td id="LC122" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    non egestas vitae, hendrerit sed quam. Phasellus a nunc eu erat varius imperdiet. Etiam id</span></td>

File: bench/snippets/_fixture.txt [hunk 4]
```diff
@@ -1304,7 +1304,7 @@
       </tr>
       <tr>
         <td id="L127" class="blob-num js-line-number" data-line-number="127"></td>
-        <td id="LC127" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    Etiam dictum tellus ipsum, nec varius quam ornare vel. Cras vehicula diam nec sollicitudin</span></td>
+        <td id="LC127" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    Etiam dictum tellus ipsum, nec various quam ornare vel. Cras vehicula diam nec sollicitudin</span></td>
```
Comment (stavares843) [inline bench/snippets/_fixture.txt]:
```suggestion
        <td id="LC127" class="blob-code blob-code-inner js-file-line"><span class="pl-s">    Etiam dictum tellus ipsum, nec varius quam ornare vel. Cras vehicula diam nec sollicitudin</span></td>

File: bench/snippets/_fixture.txt [hunk 5]
```diff
@@ -1336,7 +1336,7 @@
...
... rutrum metus convallis quis. Nullam non arcu\n    dictum, lobortis erat quis, rhoncus est. Suspendisse venenatis, mi sed venenatis vehicula,\n    tortor dolor egestas lectus, et efficitur turpis odio non augue. Integer velit sapien, dictum\n    non egestas vitae, hendrerit sed quam. Phasellus a nunc eu erat various imperdiet. Etiam id\n    sollicitudin turpis, vitae molestie orci. Quisque ornare magna quis metus rhoncus commodo.\n    Phasellus non mauris velit.\n&amp;lt;&amp;#x2f;p&amp;gt;\n&amp;lt;p&amp;gt;\n    Etiam dictum tellus ipsum, nec various quam ornare vel. Cras vehicula diam nec sollicitudin\n    ultricies. Pellentesque rhoncus sagittis nisl id facilisis. Nunc viverra convallis risus ut\n    luctus. Aliquam vestibulum &amp;lt;b&amp;gt;efficitur massa&amp;lt;&amp;#x2f;b&amp;gt;, id tempus nisi posuere a. Aliquam scelerisque\n    elit justo. Nullam a ante felis. Cras vitae lorem eu nisi feugiat hendrerit. Maecenas vitae\n    suscipit leo, lacinia dignissim lacus. Sed eget volutpat mi. In eu bibendum neque. Pellentesque\n    finibus velit a fermentum rhoncus. Maecenas leo purus, eleifend eu lacus a, condimentum sagittis\n    justo.\n&amp;lt;&amp;#x2f;p&amp;gt;&quot;;</td>
```
Comment (stavares843) [inline bench/snippets/_fixture.txt]:
```suggestion
        <td id="LC135" class="blob-code blob-code-inner js-file-line">    let string_long_escaped = &quot;\n    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris consequat tellus sit\n    amet ornare fermentum. Etiam nec erat ante. In at metus a orci mollis scelerisque.\n    Sed eget ultrices turpis, at sollicitudin erat. Integer hendrerit nec magna quis\n    venenatis. Vivamus non dolor hendrerit, vulputate velit sed, varius nunc. Quisque\n    in pharetra mi. Sed ullamcorper nibh malesuada commodo porttitor. Ut scelerisque\n    sodales felis quis dignissim. Morbi aliquam finibus justo, sit amet consectetur\n    mauris efficitur sit amet. Donec posuere turpis felis, eu lacinia magna accumsan\n    quis. Fusce egestas lacus vel fermentum tincidunt. Phasellus a nulla eget lectus\n    placerat commodo at eget nisl. Fusce cursus dui quis purus accumsan auctor.\n    Donec iaculis felis quis metus consectetur porttitor.\n&amp;lt;p&amp;gt;\n    Etiam nibh mi, &amp;lt;b&amp;gt;accumsan&amp;lt;&amp;#x2f;b&amp;gt; quis purus sed, posuere fermentum lorem. In pulvinar porta\n    maximus. Fusce tincidunt lacinia tellus sit amet tincidunt. Aliquam lacus est, pulvinar\n    non metus a, &amp;lt;b&amp;gt;facilisis&amp;lt;&amp;#x2f;b&amp;gt; ultrices quam. Nulla feugiat leo in cursus eleifend. Suspendisse\n    eget nisi ac justo sagittis interdum id a ipsum. Nulla mauris justo, scelerisque ac\n    rutrum vitae, consequat vel ex.\n&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;&amp;lt;&amp;#x2f;p&amp;gt;\n&amp;lt;p&amp;gt;\n    Sed sollicitudin &amp;lt;b&amp;gt;sem&amp;lt;&amp;#x2f;b&amp;gt; mauris, at rutrum nibh egestas vel. Ut eu nisi tellus. Praesent dignissim\n    orci elementum, mattis turpis eget, maximus ante. Suspendisse luctus eu felis a tempor. Morbi\n    ac risus vitae sem molestie ullamcorper. Curabitur ligula augue, sollicitudin quis maximus vel,\n    facilisis sed nibh. Aenean auctor magna sem, id rutrum metus convallis quis. Nullam non arcu\n    dictum, lobortis erat quis, rhoncus est. Suspendisse venenatis, mi sed venenatis vehicula,\n    tortor dolor egestas lectus, et efficitur turpis odio non augue. Integer velit sapien, dictum\n    non egestas vitae, hendrerit sed quam. Phasellus a nunc eu erat varius imperdiet. Etiam id\n    sollicitudin turpis, vitae molestie orci. Quisque ornare magna quis metus rhoncus commodo.\n    Phasellus non mauris velit.\n&amp;lt;&amp;#x2f;p&amp;gt;\n&amp;lt;p&amp;gt;\n    Etiam dictum tellus ipsum, nec varius quam ornare vel. Cras vehicula diam nec sollicitudin\n    ultricies. Pellentesque rhoncus sagittis nisl id facilisis. Nunc viverra convallis risus ut\n    luctus. Aliquam vestibulum &amp;lt;b&amp;gt;efficitur massa&amp;lt;&amp;#x2f;b&amp;gt;, id tempus nisi posuere a. Aliquam scelerisque\n    elit justo. Nullam a ante felis. Cras vitae lorem eu nisi feugiat hendrerit. Maecenas vitae\n    suscipit leo, lacinia dignissim lacus. Sed eget volutpat mi. In eu bibendum neque. Pellentesque\n    finibus velit a fermentum rhoncus. Maecenas leo purus, eleifend eu lacus a, condimentum sagittis\n    justo.\n&amp;lt;&amp;#x2f;p&amp;gt;&quot;;</td>

File: packages/bun-error/bun-error.css [hunk 6]
```diff
@@ -346,7 +346,7 @@
   -webkit-user-select: none;
 }
 
-.BunError-divet {
+.BunError-divot {
```
Comment (stavares843) [inline packages/bun-error/bun-error.css]:
```suggestion
.BunError-divet
```

File: packages/bun-types/globals.d.ts [hunk 7]
```diff
@@ -1044,7 +1044,7 @@ declare class Request implements BlobInterface {
    * Returns the referrer of request. Its value can be a same-origin URL
    * if explicitly set in init, the empty string to indicate no referrer,
    * and "about:client" when defaulting to the global's default. This is
-   * used during fetching to determine the value of the `Referer` header
+   * used during fetching to determine the value of the `Referrer` header
```
Comment (stavares843) [inline packages/bun-types/globals.d.ts]:
```suggestion
   * used during fetching to determine the value of the `Referer` header
```

File: packages/bun-types/http.d.ts [hunk 8]
```diff
@@ -88,7 +88,7 @@ declare module "http" {
     "proxy-authorization"?: string | undefined;
     "public-key-pins"?: string | undefined;
     range?: string | undefined;
-    referer?: string | undefined;
+    referrer?: string | undefined;
```
Comment (stavares843) [inline packages/bun-types/http.d.ts]:
```suggestion
    referer?: string | undefined;
```

File: packages/bun-types/http.d.ts [hunk 9]
```diff
@@ -508,7 +508,7 @@ declare module "http" {
      * header name:
      *
      * * Duplicates of `age`, `authorization`, `content-length`, `content-type`,`etag`, `expires`, `from`, `host`, `if-modified-since`, `if-unmodified-since`,`last-modified`, `location`,
-     * `max-forwards`, `proxy-authorization`, `referer`,`retry-after`, `server`, or `user-agent` are discarded.
+     * `max-forwards`, `proxy-authorization`, `referrer`,`retry-after`, `server`, or `user-agent` are discarded.
```
Comment (stavares843) [inline packages/bun-types/http.d.ts]:
```suggestion
     * `max-forwards`, `proxy-authorization`, `referer`,`retry-after`, `server`, or `user-agent` are discarded.
```

File: src/bun.js/bindings/bindings.zig [hunk 10]
```diff
@@ -1239,7 +1239,7 @@ pub const FetchHeaders = opaque {
         ProxyAuthorization,
         Purpose,
         Range,
-        Referer,
+        Referrer,
```
Comment (stavares843) [inline src/bun.js/bindings/bindings.zig]:
```suggestion
        Referer,
```

File: src/deps/diffz/DiffMatchPatch.zig [hunk 11]
```diff
@@ -420,7 +420,7 @@ fn diffHalfMatchInternal(
 
 /// Find the 'middle snake' of a diff, split the problem in two
 /// and return the recursively constructed diff.
-/// See Myers 1986 paper: An O(ND) Difference Algorithm and Its Variations.
+/// See Myers 1986 paper: An O(AND) Difference Algorithm and Its Variations.
```
Comment (stavares843) [inline src/deps/diffz/DiffMatchPatch.zig]:
```suggestion
/// See Myers 1986 paper: An O(ND) Difference Algorithm and Its Variations.
```

File: src/deps/diffz/DiffMatchPatch.zig [hunk 12]
```diff
@@ -2044,7 +2044,7 @@ test diff {
     try testing.expectEqualDeep(diffs.items, (try this.diff(arena.allocator(), "ABCDa=bcd=efghijklmnopqrsEFGHIJKLMNOefg", "a-bcd-efghijklmnopqrs", false)).items); // diff: Overlap #3.
 
     diffs.items.len = 0;
-    try diffs.appendSlice(arena.allocator(), &.{ Diff.init(.insert, " "), Diff.init(.equal, "a"), Diff.init(.insert, "nd"), Diff.init(.equal, " [[Pennsylvania]]"), Diff.init(.delete, " and [[New") });
+    try diffs.appendSlice(arena.allocator(), &.{ Diff.init(.insert, " "), Diff.init(.equal, "a"), Diff.init(.insert, "and"), Diff.init(.equal, " [[Pennsylvania]]"), Diff.init(.delete, " and [[New") });
```
Comment (stavares843) [inline src/deps/diffz/DiffMatchPatch.zig]:
```suggestion
    try diffs.appendSlice(arena.allocator(), &.{ Diff.init(.insert, " "), Diff.init(.equal, "a"), Diff.init(.insert, "nd"), Diff.init(.equal, " [[Pennsylvania]]"), Diff.init(.delete, " and [[New") });
```

File: src/js_lexer/identifier.zig [hunk 13]
```diff
@@ -379,7 +379,7 @@ pub const JumpTableInline = struct {
 //     \\Can contain IdentifierStart + Unicode 6.2 categories:
 //     \\Non-spacing mark (Mn),
 //     \\Combining spacing mark (Mc),
-//     \\Decimal number (Nd),
+//     \\Decimal number (And),
```
Comment (stavares843) [inline src/js_lexer/identifier.zig]:
```suggestion
//     \\Decimal number (Nd),
```

File: src/js_parser.zig [hunk 14]
```diff
@@ -1954,7 +1954,7 @@ const InvalidLoc = struct {
 
     pub const Tag = enum {
         spread,
-        parenthese,
+        parentheses,
```
Comment (stavares843) [inline src/js_parser.zig]:
```suggestion
        parenthese,
```

File: src/js_parser.zig [hunk 15]
```diff
@@ -1965,7 +1965,7 @@ const InvalidLoc = struct {
         @setCold(true);
         const text = switch (loc.kind) {
             .spread => "Unexpected trailing comma after rest element",
-            .parenthese => "Unexpected parentheses in binding pattern",
+            .parentheses => "Unexpected parentheses in binding pattern",
```
Comment (stavares843) [inline src/js_parser.zig]:
```suggestion
            .parenthese => "Unexpected parentheses in binding pattern",
```

File: src/js_parser.zig [hunk 16]
```diff
@@ -5564,7 +5564,7 @@ fn NewParser_(
                     if (ex.is_parenthesized) {
                         invalid_loc.append(.{
                             .loc = p.source.rangeOfOperatorBefore(expr.loc, "(").loc,
-                            .kind = .parenthese,
+                            .kind = .parentheses,
```
Comment (stavares843) [inline src/js_parser.zig]:
```suggestion
                            .kind = .parenthese,
```

File: src/js_parser.zig [hunk 17]
```diff
@@ -5601,7 +5601,7 @@ fn NewParser_(
                     }
 
                     if (ex.is_parenthesized) {
-                        invalid_loc.append(.{ .loc = p.source.rangeOfOperatorBefore(expr.loc, "(").loc, .kind = .parenthese }) catch unreachable;
+                        invalid_loc.append(.{ .loc = p.source.rangeOfOperatorBefore(expr.loc, "(").loc, .kind = .parentheses }) catch unreachable;
```
Comment (stavares843) [inline src/js_parser.zig]:
```suggestion
                        invalid_loc.append(.{ .loc = p.source.rangeOfOperatorBefore(expr.loc, "(").loc, .kind = .parenthese }) catch unreachable;
```

File: src/test/fixtures/exports-bug.js [hunk 18]
```diff
@@ -1148,7 +1148,7 @@ function WebGLRenderer(parameters) {
     if (visible) {
       if (object.isGroup) {
         groupOrder = object.renderOrder;
-      } else if (object.isLOD) {
+      } else if (object.isLOAD) {
```
Comment (stavares843) [inline src/test/fixtures/exports-bug.js]:
```suggestion
      } else if (object.isLOD) {
```

File: packages/bun-error/bun-error.css [hunk 19]
```diff
@@ -346,7 +346,7 @@
   -webkit-user-select: none;
 }
 
-.BunError-divet {
+.BunError-divet
```
Comment (stavares843) [inline packages/bun-error/bun-error.css]:
```suggestion
.BunError-divet {
```

File: bench/snippets/_fixture.txt [hunk 20]
```diff
@@ -93,7 +93,7 @@
...
+  <meta name="octolytics-dimension-user_id" content="2625545" /><meta name="octolytics-dimension-user_login" content="botika" /><meta name="octolytics-dimension-repository_id" content="160065385" /><meta name="octolytics-dimension-repository_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_public" content="true" /><meta name="octolytics-dimension-repository_is_fork" content="false" /><meta name="octolytics-dimension-repository_network_root_id" content="160065385" /><meta name="octolytics-dimension-repository_network_root_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false" />```
```
Comment (stavares843) [inline bench/snippets/_fixture.txt:96]:
```suggestion
  <meta name="octolytics-dimension-user_id" content="2625545" /><meta name="octolytics-dimension-user_login" content="botika" /><meta name="octolytics-dimension-repository_id" content="160065385" /><meta name="octolytics-dimension-repository_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_public" content="true" /><meta name="octolytics-dimension-repository_is_fork" content="false" /><meta name="octolytics-dimension-repository_network_root_id" content="160065385" /><meta name="octolytics-dimension-repository_network_root_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false" />```
```suggestion
  <meta name="octolytics-dimension-user_id" content="2625545" /><meta name="octolytics-dimension-user_login" content="botika" /><meta name="octolytics-dimension-repository_id" content="160065385" /><meta name="octolytics-dimension-repository_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_public" content="true" /><meta name="octolytics-dimension-repository_is_fork" content="false" /><meta name="octolytics-dimension-repository_network_root_id" content="160065385" /><meta name="octolytics-dimension-repository_network_root_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false" />
```
```

Comment (stavares843) [inline bench/snippets/_fixture.txt]:
```suggestion
  <meta name="octolytics-dimension-user_id" content="2625545" /><meta name="octolytics-dimension-user_login" content="botika" /><meta name="octolytics-dimension-repository_id" content="160065385" /><meta name="octolytics-dimension-repository_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_public" content="true" /><meta name="octolytics-dimension-repository_is_fork" content="false" /><meta name="octolytics-dimension-repository_network_root_id" content="160065385" /><meta name="octolytics-dimension-repository_network_root_nwo" content="botika/v_escape" /><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false" />
```

File: misctools/gen-unicode-table.js [hunk 21]
```diff
@@ -31,7 +31,7 @@ const idStartES5 = []
 
 // UnicodeCombiningMark: any character in the Unicode categories “Non-spacing mark (Mn)”
 // or “Combining spacing mark (Mc)”
-// UnicodeDigit: any character in the Unicode category “Decimal number (Nd)”
+// UnicodeDigit: any character in the Unicode category “Decimal number (And)”
```
Comment (stavares843) [inline misctools/gen-unicode-table.js]:
```suggestion
// UnicodeDigit: any character in the Unicode category “Decimal number (Nd)”
```

**Comment by Jarred-Sumner** [comment]
Thanks for the PR

**Comment by Jarred-Sumner** [comment]
Will need to do some manual testing to make sure it doesn't cause any tests to break

**Comment by stavares843** [comment]
changes were discarding when updating PR, i can add a new one

**Pull request closed without merging**